modules/general/acs-email-with-config-publish/README.md

```diff
@@ -0,0 +1,42 @@
+# acs-email-managed-domain-with-published-config
+
+Azure Communication Email Service with managed Azure domain, with published configuration
+
+## Description
+
+This module provisions a top-level Azure Communication Service (ACS) resource and an Email Communication resource with a managed Azure domain, linking the domain to the ACS resource. A send-from email address is also configured for the domain, which defaults to `DoNotReply@<domain>`.
+
+The ACS key, ACS connection string, the email domain, and the send-from email address are published into the specified Key Vault.
```

Comment:
Should we surface these settings in AppConfiguration too?

Reply:
I did initially have the non-secret values in AppConfiguration and the secret values in KV. Then I realised that we can only easily access the KV ones from Synapse so I put everything in there.

We can publish the values into AppConfiguration (backed by KV secrets) as part of the `synapse-enterprise` module, which is what we do for the Application Insights secrets.

Do you think we should also be publishing to AppConfiguration here though?

Reply:
We have integrated Synapse with App Config in projects of past, and I don't believe it was too gnarly. In case that helps this decision.

Reply:
I think having them backed by KV secrets is fine (seems unnecessary to store twice).

Looking at other modules we're not as consistent as I thought :-)  It's probably a separate task to do a pass through all modules and add the KV references to App Config.